### ISSUE TITLE
[Request] Checkpoints of other methods

### ISSUE THREAD

**Issue by kavisha725** [jac99/Egonn#3]
Hi, 

Thank you for releasing your code and checkpoints. I was able to recreate the results in the paper. This is not an issue but I couldn't find your contact details so I thought of posting here. 

I was wondering if you could also release the checkpoints of other methods (eg. MinkLoc3D, DiSCO and DH3D). 

Kind regards,
Kavisha

**Comment by jac99** [comment]
Hi,
Unfortunately, I cannot find checkpoints of other methods. Experiments were made some time ago, and checkpoints are lost.
However, you should be able to train MinkLoc3D on the same training datasets as EgoNN method (Mulran + Apollo SouthBay). 
You'll need to modify the training code in `trainer.py`. For EgoNN, in one training iteration, first, the loss for the global descriptor is calculated, then the loss for the local descriptor. You'll need to modify this code to compute only the loss for the global descriptor (as MinkLoc3D does not compute local descriptors). And remove everything connected with local descriptor loss. 
Then, instead of using egonn.txt model_config use minkloc3d_mulran.txt model_config (in the `models` subfolder) when training the model:
`python train.py --config ../config/config_egonn.txt --model_config ../models/minkloc3d_mulran.txt`

**Comment by kavisha725** [comment]
No worries. Thank you for the instructions. 
Closing the issue.
